docs/contributing/translation/sync-guide.md

```diff
@@ -17,3 +46,126 @@ git pull source master
 ```
 
 Fix all [merge conflicts](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line) and create a pull request to finish the merge.
+
+## Common types of merge issues
+
+### Frontmatter changes
+
+This type of conflict reflects a change to the [frontmatter]() of a document.
+
+Removing a field:
+
+```diff
++ <<<<<<<
+title: my stub article
+issue: #12345
++ =======
++ title: My Stub Article
++ >>>>>>>
+```
+
+Adding a field:
+
+```diff
++ <<<<<<<
+title: Documentacion
++ =======
++ title: Docs
++ description: The one-stop shop for documentation in Gatsby!
++ >>>>>>>
+```
+
+Changing a field:
+
+```diff
++ <<<<<<<
+title: My old title
++ =======
++ title: My new title
++ >>>>>>>
+```
+
+### Typos fixes
+
+Sometimes there is a typo or grammatical error in the English source that gets fixed in an update.
+
+```diff
+- Let's all use Gabsty!
++ Let's all use Gatsby!
+```
+
+```diff
+- Gatsby is awesome framework!
++ Gatsby is an awesome framework!
+```
+
+Since these typos most likely don't exist in the translated version, you can most likely use the translated version as-is.
+
+### Formatting changes
+
+Sometimes the only difference is a change in formatting, such as adding back-ticks to annotate something as code, or changing the level of headings:
+
+```diff
+- Sass files have the extension .scss.
++ Sass files have the extension `.scss`.
+```
+
+```diff
+- #### Formatting changes
++ ### Formatting changes
+```
+
+The only necessary change is to ensure the translated content carries over these formatting updates.
+
+### Link updates
+
+These changes involve updating the URL of a link:
+
+```diff
+- Please see our [plugins page](/packages).
++ Please see our [plugins page](/plugins).
+```
+
+```diff
+- Check out our [GitHub](http://github.com/gatsbyjs/gatsby)
++ Check out our [GitHub](https://github.com/gatsbyjs/gatsby)
+```
+
+Resolve these changes by updating the link.
+
+### Content changes
+
+Sometimes, the content of the source page is actually updated and needs a translation. Make sure to read the change carefully and change the translation to match its meaning.
+
+### Conflicts in untranslated files
+
+Sometimes, you may find conflicts in files that haven't been translated yet:
+
+```diff
++ <<<<<<<
+This is old English content.
++ =======
++ This is new English content.
++ >>>>>>>
+```
+
+This is usually because of an previous improper merge (for example, using the "Squash and merge" option). In this case, it's usually alright to accept the new content. In VS Code, this is done using the "Accept Incoming Change" option:
```

Comment:
```suggestion
This is usually because of a previous improper merge (for example, using the "Squash and merge" option). In this case, it's usually alright to accept the new content. In VS Code, this is done using the "Accept Incoming Change" option:
```